### ISSUE TITLE
Migration from 1.11 to 1.12 failing

### ISSUE THREAD

**Issue by cromerc** [go-gitea/gitea#11984]
- Gitea version (or commit ref): 1.12.0
- Git version: 2.20.1
- Operating system: Debian Buster
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [X] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [X] Not relevant
- Log gist:
https://gist.github.com/cromerc/e1bcde4a00f5820fef4bff0fbfa08670

## Description
Migration is failing for me under MySQL and now gitea doesn't start.
...

**Comment by philippbeckmann** [comment]
Please have a look at Issue https://github.com/go-gitea/gitea/issues/9347 and the documentation https://docs.gitea.io/en-us/command-line/#convert

**Comment by cromerc** [comment]
Thanks. The convert command line option fixed the database. Then I restarted gitea 1.12.0, it used up all the available memory on my VPS causing the server to lock up and for me to loose ssh access. I ended up having to reboot my VPS into rescue mode, disable gitea, and reboot the server. After that I tried running gitea again and now it works. I have no idea what happened during the first run of gitea 1.12.0... but it killed my server. But it looks like everything is working now.
